### ISSUE TITLE
Crash on asm transformation

### ISSUE THREAD

**Issue by Caellian** [Chicken-Bones/CodeChickenLib#35]
I have a clean (Java 1.8) forge (decompiled) workspace with only
IC2 2.2.660 experimental api, CodeChickenCore-1.8-1.0.5.34-dev, NotEnoughItems-1.8-1.0.5.82-dev
and my mod installed. I'm in a process of updating my mod and it does crash a lot, but these crashes aren't related to my mod in any way AFAIK. I can't get around minecraft crashing in any way other than replacing your mods with src versions of them, but I'm using your NEI api to hide items from NEI.

So basically, here's the crash log: http://pastebin.com/CCzwyt7b

I have manually extracted lib jar (CodeChickenLib-1.8-1.1.2.115-dev) from CodeChickenCore-1.8-1.0.5.34-dev jar because I was getting another crash: http://pastebin.com/hJ5jXpxq

Just for reference, I really need the api because I'm adding custom crafting and I'm going to add custom crafting guides for NEI.

**Comment by Chicken-Bones** [comment]
Forge version?

**Comment by GirafiStudios** [comment]
Looks like it's .1289 :)

**Comment by Feronzed** [comment]
It was, updated to forge 1290 and it still crashes. (Should I downgrade?)
Also, I forgot to mention that I disabled asm and almost all features except for creepers not blowing up terrain.

Here are my config files: http://pastebin.com/cTzPpGaV

Also, the crash report doesn't get generated. (That's why I posted a log)

**Comment by Feronzed** [comment]
1297, still here...

**Comment by 61352151511** [comment]
Same issue happening with Latest forge 1.7.10 versions for all forge, codechickenlib, codechickencore, and NEI. I as well have java 8, I tried backdating codechickenlib to the first 1.7.10 version as well and get the same problem. Not sure if it's a java 8 problem or something that was put in forge, no idea.
